bazel/browsers/chromium/chromium.bzl

```diff
@@ -21,6 +21,11 @@ def define_chromium_repositories():
         named_files = {
             "CHROMIUM": "chrome-linux/chrome",
         },
+        exclude_patterns = [
+            # Exclude a log file that chromium writes to each run, causing remote cache
+            # misses in downstream targets.
+            "chrome-linux/chrome_debug.log",
```

Comment:
Do we need a similar thing for e.g. Mac or Windows?

Reply:
I'm not sure. I'll test both and report back.

Reply:
@devversion It doesn't appear to cause cache misses on Windows.

On OSX, it's a bit more complicated. Using bazel-provided chromium in unsandboxed execution fails early because it can't build runfiles due to all of the files with spaces in them. If I exclude all of those files, then the runfiles can be built but chromium fails, probably because it needs some of those files.

It's not clear to me why it works with sandboxed execution on OSX but fails when run outside of the sandbox. My understanding was that using `--nobuild_runfile_links` fixed the runfiles with spaces issue but that doesn't seem to work for unsandboxed execution.